experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -191,7 +192,10 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 			}
 
 			if (change !== undefined) {
-				this.changeEvents.emit("newLocalState", this.changeFamily.intoDelta(change));
+				this.changeEvents.emit(
+					"newLocalState",
+					this.changeFamily.intoDelta(change, () => EmptyKey),
```

Comment:
Isn't this a problem? If that delta is used on a forest it will not create the correct repair data.

Reply:
Shouldn't this use the localBranch's RepairDataHandler?

Reply:
This will be removed right after this PR, so that's OK.